src/gui/book/Book.cpp

```diff
@@ -382,7 +377,7 @@ void PlayerBook::openPage(ARX_INTERFACE_BOOK_MODE newPage, bool toggle) {
 
 	} else {
 		// Otherwise open the book
-		ARX_INTERFACE_BookToggle();
+		this->toggle();
```

Comment:
Unnecessary `this->`.

Reply:
Not really, the member function's parameter is also named `toggle`. Shall I rename it?

Reply:
Yeah, renaming would be good.